docs/docs-ref-autogen/excel/excel/excel.eventtype.yml

```diff
@@ -239,7 +246,7 @@ fields:
     package: excel!
     summary: >-
       WorksheetFormulaChanged represents the type of event registered on a
-      worksheet, and occurs when a formula is changed.
```

Comment:
This change should be reverted.

Reply:
@Rick-Kirkham Can you make sure this is fixed in DefinitelyTyped? Thx